### PR TITLE
kitty: add zap stanza

### PR DESCRIPTION
**Pull request by MindTooth**
Add zap stanza to kitty.

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/kitty.rb [hunk 1]
```diff
@@ -9,4 +9,11 @@
   homepage 'https://github.com/kovidgoyal/kitty'
 
   app 'kitty.app'
+  binary '/Applications/kitty.app/Contents/Frameworks/kitty/terminfo', target: "#{ENV['HOME']}/.terminfo"
```
Comment (commitay) [inline Casks/kitty.rb]: Remove this

File: Casks/kitty.rb [hunk 2]
```diff
@@ -9,4 +9,11 @@
...
+
+  zap trash: [
+               '~/.config/kitty', # Standard XDG_CONFIG_HOME path
```
Comment (commitay) [inline Casks/kitty.rb]: Remove the comment

File: Casks/kitty.rb (hunk 1 above)
Comment (MindTooth) [reply Casks/kitty.rb]: How do I then fix the issue at hand?

Comment (commitay) [reply Casks/kitty.rb]: Adding this would be a workaround, not a fix.

**Pull request merged**